fastlane/Fastfile

```diff
@@ -866,49 +865,85 @@ platform :ios do
   end
 
   desc "Build and deploy PurchaseTesterSwiftUI"
-  lane :deploy_purchase_tester do
-    latest_changelog = File.read("#{File.dirname(__FILE__)}/../#{changelog_latest_path}")
+  lane :deploy_purchase_tester do |options|
+    dry_run = options[:dry_run] || false
+
+    project_root = File.expand_path("#{File.dirname(__FILE__)}/..")
+
+    latest_changelog = File.read("#{project_root}/#{changelog_latest_path}")
     changelog = "Release #{current_version_number}:\n\n#{latest_changelog}"
 
-    def ios_build(changelog)
+    purchase_tester_root = "#{project_root}/Tests/TestingApps/PurchaseTesterSwiftUI"
+    xcodeproj_path = "#{purchase_tester_root}/PurchaseTester.xcodeproj"
+
+    # The following commands use configuration that lives in the PurchaseTesterSwiftUI directory, 
+    # like the Matchfile. So we move to that project's directory to run them.
+    Dir.chdir("#{purchase_tester_root}/fastlane") do
+      match(readonly: true, platform: "ios")
       increment_build_number(
         build_number: latest_testflight_build_number(platform: 'ios') + 1,
-        xcodeproj: 'PurchaseTester.xcodeproj'
+        xcodeproj: xcodeproj_path,
       )
+    end
+    
+    build_ios_app(
+      workspace: 'RevenueCat.xcworkspace',
+      scheme: "PurchaseTester",
+      destination: "generic/platform=ios",
+      xcodebuild_formatter: '',
+      export_method: 'app-store',
+      export_options: {
+        manageAppVersionAndBuildNumber: true,
+        provisioningProfiles: {
+          "com.revenuecat.sampleapp" => "match AppStore com.revenuecat.sampleapp",
+          "com.revenuecat.sampleapp.watchkitapp" => "match AppStore com.revenuecat.sampleapp.watchkitapp"
+        }
```

Comment:
I ended up passing the profiles explicitly here. I'm not entirely sure what I'm missing, but it seems like fastlane was reading the values from the Matchfile previously. I don't think there's a way for fastlane to read the Matchfile from a different directory. 

Maybe @joshdholtz knows? Maybe I could just import it?